osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/EventLogCategoryDto.java

```diff
@@ -12,8 +12,7 @@
     FRAUD_DETECTION_LOG(2),
     COMMUNICATION_SESSION_LOG(3),
     M_BUS_EVENT_LOG(2),
-    POWER_QUALITY_EVENT_LOG(2),
-    POWER_FAILURE_EVENT_LOG(2);
```

Comment:
Shouldn't it be possible to get this type of events with the FindEvents request?